examples/Esp32-ssd1306/Esp32-ssd1306.ino

```diff
@@ -14,128 +14,78 @@
  *****************************************************************************/
 #include <WiFiClientSecure.h>
 #define NO8266
```

Comment:
If we can find a way to detect ESP8266 and avoid this #define, would be useful.

Reply:
As far as I can remember we can do something like the bellow. I'll try later at home. 
```
#if defined(ESP8266)
#define SOMETHING HERE
#endif
```

Reply:
SGTM!